src/params.ts

```diff
@@ -0,0 +1,12 @@
+import { RouteParams } from 'vue-router'
+import { RouterMock } from './router'
+import { getRouter } from './plugin'
+
+/**
+ * Sets the params of the current route
+ * @param params - params to set in the current route
+ */
+export function setParams(params: RouteParams) {
+  const router: RouterMock = getRouter()
+  router.currentRoute.value.params = params
```

Comment:
when modifying the `currentRoute.value` it should be set as a whole to keep it consistent with vue router

Reply:
Do you mean something like `router.currentRoute.value = { ...router.currentRoute.value, params }`?